server/utils/features.js

```diff
@@ -60,7 +60,13 @@ const features = {
     )
   ),
   pastCasesNavigation: enabledForAll(),
-  caseProgressNextAppearanceBadge: enabledForSourceTypes('COMMON_PLATFORM', 'LIBRA')
+  caseProgressNextAppearanceBadge: enabledForSourceTypes('COMMON_PLATFORM', 'LIBRA'),
+  hearingOutcomes: anyOf(
```

Comment:
just curious if this needs to be nested? you've got `anyOf` with a single `allOf' (being new to this not entirely sure how it works)

Reply:
You've spotted an error - I was thinking of doing pre-prod too and then removed it - and below I want to remove pre-prod for now! Thanks